acs-core/src/main/java/com/arrow/acs/client/model/CompanyModel.java

```diff
@@ -10,6 +10,9 @@
  *******************************************************************************/
 package com.arrow.acs.client.model;
 
+import java.util.HashSet;
+import java.util.Set;
+
 public class CompanyModel extends AuditableDocumentModelAbstract<CompanyModel> {
 	private static final long serialVersionUID = -1173608869956123803L;
```

Comment:
In general, it is desired to re-generate serialVersionUID if change class fields